sql/src/test/java/io/crate/execution/engine/distribution/merge/RamAccountingPageIteratorTest.java

```diff
@@ -104,17 +93,28 @@ public void testWrappingApplied() {
         assertThat(((RamAccountingPageIterator) pagingIterator1).delegatePagingIterator,
                    instanceOf(PassThroughPagingIterator.class));
 
-        PagingIterator<Integer, Row> pagingIterator2 = PagingIterator.create(
+        PagingIterator<Integer, Row> repeatingSortedPagingIterator = PagingIterator.create(
             2,
             true,
             PositionalOrderBy.of(
                 new OrderBy(Collections.singletonList(Literal.of(1)), new boolean[] {false}, new Boolean[] {false}),
                 Collections.singletonList(Literal.of(1))),
             () -> null);
 
-        assertThat(pagingIterator2, instanceOf(RamAccountingPageIterator.class));
-        assertThat(((RamAccountingPageIterator) pagingIterator2).delegatePagingIterator,
+        assertThat(repeatingSortedPagingIterator, instanceOf(RamAccountingPageIterator.class));
+        assertThat(((RamAccountingPageIterator) repeatingSortedPagingIterator).delegatePagingIterator,
                    instanceOf(SortedPagingIterator.class));
+
+        PagingIterator<Integer, Row> nonRepeatingSortedPagingIterator = PagingIterator.create(
+            2,
+            false,
+            PositionalOrderBy.of(
+                new OrderBy(Collections.singletonList(Literal.of(1)), new boolean[] {false}, new Boolean[] {false}),
```

Comment:
could you still mock the PositionalOrderBy?

Reply:
I guess we could. But why?

Reply:
because it's irrelevant to the actual test here